iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/expected.inl

```diff
@@ -277,7 +282,7 @@ inline expected<ErrorType>::expected(error<ErrorType>&& errorValue) noexcept
     : m_store(in_place_index<ERROR_INDEX>(), std::move(errorValue.value))
 {
 }
-
+// AXIVION DISABLE STYLE AutosarC++19_03-A16.0.1: conditional compilation is required for setting gcc diagnostics
```

Comment:
I do not understand the justification at all. What is the relation between the windows platform and gcc?

Reply:
@elfenpiff corrected the justification